homeassistant/components/mystrom/sensor.py

```diff
@@ -44,13 +50,40 @@ class MyStromSwitchSensorEntityDescription(SensorEntityDescription):
         native_unit_of_measurement=UnitOfPower.WATT,
         value_fn=lambda device: device.consumption,
     ),
+    MyStromSwitchSensorEntityDescription(
+        key="energy_since_boot",
+        translation_key="energy_since_boot",
+        device_class=SensorDeviceClass.ENERGY,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
+        value_fn=lambda device: None
+        if device.energy_since_boot is None
+        else device.energy_since_boot / 3600000,
+    ),
     MyStromSwitchSensorEntityDescription(
         key="temperature",
         device_class=SensorDeviceClass.TEMPERATURE,
         state_class=SensorStateClass.MEASUREMENT,
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         value_fn=lambda device: device.temperature,
     ),
+    MyStromSwitchSensorEntityDescription(
+        key="time_since_boot",
+        translation_key="time_since_boot",
+        device_class=SensorDeviceClass.DURATION,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        native_unit_of_measurement=UnitOfTime.SECONDS,
+        value_fn=lambda device: device.time_since_boot,
+        suggested_display_precision=0,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
+    MyStromSwitchSensorEntityDescription(
+        key="boot_id",
+        translation_key="boot_id",
+        icon="mdi:identifier",
+        value_fn=lambda device: device.boot_id,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
```

Comment:
What's a boot id?

Reply:
I think that's a meh usecase, as in, what would you use that for? In theory we can use the device class TOTAL_INCREASING, and in the end the user doesn't really care when that happens as it just keeps counting, so I am kinda missing the end use case for this :)